docs/src/main/sphinx/connector/hive.rst

```diff
@@ -968,5 +968,3 @@ Hive 3 related limitations
       )
       AS <query>
 
-  Trino does not support gathering table statistics for Hive transactional tables.
-  You need to use Hive to gather table statistics with ``ANALYZE TABLE COMPUTE STATISTICS`` after table creation.
```

Comment:
We may want to keep a mention that statisitcs colleciton is still not supported in INSERT and CTAS flows

Reply:
Good point - - added.